spec/unit/type/file/group_spec.rb

```diff
@@ -27,6 +27,13 @@
       expect { group.insync?(5) }.to raise_error(/Could not find group foos/)
     end
 
+    it "should return false if a group's id can't be found by name in noop" do
+      Puppet[:noop] = true
+      allow(resource.provider).to receive(:name2uid).and_return(nil)
```

Comment:
```suggestion
      allow(resource.provider).to receive(:name2gid).and_return(nil)
```

because of the `Puppet[:noop]` line above I found that the allow doesn't impact the test at all, but maybe it still makes sense to keep it as it makes things easier to read